README.md

```diff
@@ -5,11 +5,1119 @@
+  <GraphQL schema>
+  <Nexus/Nexus-Prisma type defs>
+  <Prisma schema model defs>
+```
+
+#### One-Create Operation
+
+Allow clients to create records of the respective model.
+
+Relation fields may be connected with an existing record or a sub-create may be inlined. If the relation is a `List` then multiple connections or sub-creates are permitted.
+
+Each instance of this operation contributions to GraphQL Schema:
+
+- `×1` `InputObject` `<ModelName>CreateInput`
+- `×N` `InputObject` per relational field.
+
+  This is a container for the sub-create and connect (below). Cannot be amortized becuase sub-create cannot.
+
+- `×N` `InputObject` per relational field create.
+
+  Cannot be amoritized because of specialized shape excluding field where supplying relation of type of parent `Object` being created would normally be. Instead this relation being created will be intuitively connected to the `Object` being directly created.
+
+- `×N (A)` `InputObject` `<ModelName>WhereUniqueInput` per relational field connect.
+
+Example:
+
+```gql
+mutation simple {
+  createOneUser(data: { email: "newton@prisma.io" }) {
+    id
+  }
+}
+
+mutation connectRelation {
+  createOneUser(data: { email: "newton@prisma.io", posts: { connect: [43] } }) {
+    id
+  }
+}
+
+mutation createRelation {
+  createOneUser(
+    data: {
+      email: "newton@prisma.io"
+      posts: { create: [{ title: "On How The Prism Came To Be", body: "..." }] }
+    }
+  ) {
+    id
+    posts {
+      title
+    }
+  }
+}
+```
+
+```gql
+type Mutation {
+  createOneUser(data: UserCreateInput!): User!
+}
+
+type Post {
+  author: User!
+  id: Int!
+  title: String!
+  body: String!
+}
+
+input PostCreateManyWithoutPostsInput {
+  connect: [PostWhereUniqueInput!]
+  create: [PostCreateWithoutAuthorInput!]
+}
+
+input PostCreateWithoutAuthorInput {
+  title: String!
+  body: String!
+}
+
+input PostWhereUniqueInput {
+  id: Int
+  title: String
+}
+
+type User {
+  email: String!
+  id: Int!
+  posts: [Post!]!
+}
+
+input UserCreateInput {
+  email: String!
+  posts: PostCreateManyWithoutPostsInput
+}
+```
+
+```ts
+mutationType({
+  definition(t) {
+    t.crud.createOneUser()
+  },
+})
+
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.id()
+    t.model.email()
+    t.model.posts()
+  },
+})
+
+objectType({
+  name: 'Post',
+  definition(t) {
+    t.model.id()
+    t.model.title()
+    t.model.body()
+    t.model.author()
+  },
+})
+```
+
+```prisma
+model User {
+  id    Int    @id @unique
+  email String @unique
+  posts Post[]
+}
+
+model Post {
+  id     Int    @id
+  title  String @unique
+  body   String
+  author User
+}
+```
+
+#### One-Read Operation
+
+Allow clients to find a specific record of the respective model. They may search by any field in the model marked with `@unique` attribute.
+
+The ability for relation fields to be filtered sorted etc. depends on if such features have been enabled for those `Object`s [via `t.model`](/TODO).
```

Comment:
> The ability for relation fields to be filtered sorted etc.

I suggest to spell out pagination as third feature: "The ability for relation fields to be filtered, sorted and/or paginated depends on ..."